src/neo/Network/P2P/TaskManager.cs

```diff
@@ -24,6 +24,8 @@ private class Timer { }
 
         private static readonly TimeSpan TimerInterval = TimeSpan.FromSeconds(30);
         private static readonly TimeSpan TaskTimeout = TimeSpan.FromMinutes(1);
+        private static readonly UInt256 HeaderTaskHash = UInt256.Zero;
```

Comment:
Why has this become `static`?

Reply:
It will no change between different instances